File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
+//Clear the footnote content.
+footnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the footnote.
+WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+//Set the footnote character format.
+footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the footnote text.
+footnoteParagraph.AppendText(" Footnote is modified.");
+//Access paragraph in a Word document.
+paragraph = document.Sections[2].Paragraphs[1] as WParagraph;
+//Access the endnote in the paragraph.
+WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear the endnote content.
+endnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the endnote.
+WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
+//Set the endnote character format.
+endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the endnote text.
+endnoteParagraph.AppendText(" Endnote is modified.");
+//Save the the Word document to the MemoryStream.
+MemoryStream outputStream = new MemoryStream();
+document.Save(outputStream, FormatType.Docx);
+outputStream.Position = 0;
+//Download as Word document in the browser.
+return File(outputStream, "application/msword", "Sample.docx");
+  }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+//Load the file stream into a Word document.
+using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+{
+//Access paragraph in a Word document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
+//Access the footnote in the paragraph.
+WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear the footnote content.
+footnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the footnote.
+WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+//Set the footnote character format.
+footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the footnote text.
+footnoteParagraph.AppendText(" Footnote is modified.");
+//Access paragraph in a Word document.
+paragraph = document.Sections[2].Paragraphs[1] as WParagraph;
+//Access the endnote in the paragraph.
+WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear the endnote content.
+endnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the endnote.
+WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
+//Set the endnote character format.
+endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the endnote text.
+endnoteParagraph.AppendText(" Endnote is modified.");
+//Save the Word document to the MemoryStream.
+MemoryStream stream = new MemoryStream();
+document.Save(outputStream, FormatType.Docx);
```

Comment:
@sharonaumao - Please change the `MemoryStream` object naming.

Reply:
@Ramaraj-Marimuthu -Done